pkg/gatherers/conditional/gather_api_request_count.go

```diff
@@ -0,0 +1,107 @@
+package conditional
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/openshift/insights-operator/pkg/gatherers"
+	"github.com/openshift/insights-operator/pkg/record"
+	"github.com/openshift/insights-operator/pkg/utils"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+)
+
+// APIRequestCount defines a type used when marshaling into JSON
+type APIRequestCount struct {
+	ResourceName        string `json:"resource"`
+	RemovedInRelease    string `json:"removed_in_release"`
+	TotalRequestCount   int64  `json:"total_request_count"`
+	LastDayRequestCount int64  `json:"last_day_request_count"`
+}
+
+// BuildGatherApiRequestCounts creates a gathering closure which collects logs api requests counts for the
+// resources mentioned in
+// Params is of type AlertIsFiringConditionParams:
+//   - alert_name string - name of the firing alert
+//
+//
+// * Location in archive: conditional/api_request_counts.json
```

Comment:
I have to distinguish it, because I will have a second similar alert....naming is hard. What about:
`conditional/<alert_name>/api_request_counts.json` 
WDYT pls @Sergey1011010 ?

Reply:
Well I have only one alert so it's simpler, but I am still not sure this is the best location.